fusor/examples/README.md

```diff
@@ -4,6 +4,6 @@
 * `bcr_abl1_expanded.json`: Equivalent fusion to the above, but with expanded descriptions of genes, locations, and sequences provided by SeqRepo and the VICC Gene Normalizer.
 * `alk.json`: Example of an ALK fusion, demonstrating use of a categorical "multiple possible gene" component, retrieved from a human-curated database like [CIViC](https://civicdb.org/variants/499/summary). Represented in nomenclature as `ALK(hgnc:427)::v`.
 * `ewsr1.json`: An EWSR1 assayed fusion, demonstrating an assay description object and use of the "unknown gene" partner. Represented in nomenclature as `EWSR1(hgnc:3508)::?`.
-* `tpm3_ntrk1.json`: Example TPM3-NTRK1 assayed fusion drawn from previous VICC Fusion Curation draft material. Represented in nomenclature as `refseq:NM_152263.3(TPM3):e.8::refseq:NM_002529.3(NTRK1):e.10`.
-* `tpm3_pdgfrb.json`: Example TPM3-PDGFRB assayed fusion identified via RT-PCR. Represented in nomenclature as `refseq:NM_152263.3(TPM3):e.8::refseq:NM_002609.3(PDGFRB):e.11`.
+* `tpm3_ntrk1.json`: Example TPM3-NTRK1 assayed fusion drawn from previous VICC Fusion Curation draft material. Represented in nomenclature as `refseq:NM_152263.3(TPM3):e.1_8::refseq:NM_002529.3(NTRK1):e.10_22`.
```

Comment:
Why is the top one 10 but the bottom one is 11? @jsstevenson

Reply:
Just how the fusions happened, I guess. The TPM3-NTRK fusion + associated data is an example that the group has been working with for a while, and the TPM3-PDGFRB fusion was, I think, something that I pulled from a paper somewhere.

Reply:
Also, if you want to make this one have full transcript segments rather than just junctions, make sure to update the JSON file and the test_nomenclature test case as well